src/pages/AddLiquidity/index.tsx

```diff
@@ -347,6 +347,7 @@ export default function AddLiquidity({
               }}
               onCurrencySelect={handleCurrencyASelect}
               currency={currencies[Field.CURRENCY_A]}
+              maxAmount={maxAmounts[Field.CURRENCY_A]?.toExact()}
```

Comment:
Wondering should we do the toExact() conversion part inside the CurrencyInputPanel component as to not repeat it in every place?